src/components/SubmissionPage/SubmissionPage.jsx

```diff
@@ -0,0 +1,50 @@
+import { Flex, Input, Button } from "@chakra-ui/react";
+import { useRef, useState, useEffect } from "react";
+
+export const SubmissionPage = () => {
+  const [disableButton, setDisabledButton] = useState(true);
+  const inputRef = useRef();
+
+  function isUrlValid(string) {
```

Comment:
- you can make this a utility function, & add it in a utils file
- instead of `string` as a parameter name `portfolioUrl` would be more readable
```suggestion
  function isUrlValid(portfolioUrl) {
```

Reply:
Yeah done Thank you.